### PR TITLE
Hippodrome Homework

### PR DESCRIPTION
**Pull request by feeL67**

### CODE REVIEW

**Review by gr00shik — COMMENTED** [review]
not correct

File: src/main/java/com/tms/lesson30/entity/Pair.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    private Horse horse;
+    private HorseMan horseman;
+    double distance;
+
+    public Pair(Horse horse, HorseMan horseman) {
+        this.horse = horse;
+        this.horseman = horseman;
+    }
+
+    public double getDistance() {
```
[the remark this answers is not in the document]
Comment (feeL67) [reply src/main/java/com/tms/lesson30/entity/Pair.java:22]: mmm, wrong name.
rename it to couplingCoefficient.

File: src/main/java/com/tms/lesson30/services/BetServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    public void makeABet(List<Pair> pairs) {
+        System.out.println("Enter the number of Pair, you want to bet on:");
+        Scanner scanner = new Scanner(System.in);
+        int numberPair = scanner.nextInt();
+        betPair = pairs.get(numberPair - 1);
+        System.out.println("Please enter your bet");
+        bet = scanner.nextInt();
+        System.out.println("You bet on " + betPair.getHorse().getName() + " " + betPair.getHorseman().getName() + " " + bet + "$$$");
+        try {
+            Thread.sleep(2000);
```
[the remark this answers is not in the document]
Comment (feeL67) [reply src/main/java/com/tms/lesson30/services/BetServiceImpl.java:25]: remove

File: src/main/java/com/tms/lesson30/services/BetServiceImpl.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Scanner;
+
+@Service
+public class BetServiceImpl implements BetService<Pair> {
+
+    private int bet;
+    private Pair betPair;
```
[the remark this answers is not in the document]
Comment (feeL67) [reply src/main/java/com/tms/lesson30/services/BetServiceImpl.java:13]: deleted

File: src/main/java/com/tms/lesson30/services/HorseRacingServiceImpl.java [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+public class HorseRacingServiceImpl implements HorseRacingService<Pair> {
+
+    @Autowired
+    private BetService<Pair> betService;
+
+    public List<Pair> getAll() {
+        Random random = new Random();
+        List<SPEED> speedList = Arrays.asList(LOW, MEDIUM, HIGH);
+        List<LEVEL> levelList = Arrays.asList(LEVEL.LOW, LEVEL.MEDIUM, LEVEL.HIGH);
+        Pair pair1 = new Pair
```
[the remark this answers is not in the document]
Comment (feeL67) [reply src/main/java/com/tms/lesson30/services/HorseRacingServiceImpl.java:28]: done

**Pull request merged**